lib/atomic/organizations.ex

```diff
@@ -255,6 +262,24 @@ defmodule Atomic.Organizations do
     end
   end
 
+  @doc """
+  Returns all roles bigger or equal to the given role.
+
+  ## Examples
+
+      iex> roles_bigger_than_or_equal(:member)
+      [:member, :admin, :owner]
+
+  """
+  def roles_bigger_than_or_equal(role) do
+    case role do
+      :follower -> [:follower, :member, :admin, :owner]
+      :member -> [:member, :admin, :owner]
+      :admin -> [:admin, :owner]
+      :owner -> [:owner]
+    end
```

Comment:
```suggestion
    list = [:follower, :member, :admin, :owner]
    Enum.drop_while(list, fn elem -> elem != role end)
```

I am not too sure about this, but if we want to add more roles in the future, we can just insert in the list in order, and not need to do anything else to this function.